docu/release_notes/2015-11-02-Techne10.html.md

```diff
@@ -1 +1 @@

<h4>Cards</h4>
<p>Guidelines on icon shapes to you use on cards and tables.</p>

<p>Thank you everyone for participating and helping us to make this first major release happen.</p>
<p>There will be more in the future, stay tuned.</p>
\ No newline at end of file
+---
date: November 2, 2015
service: 'techne'
official_version: v1.0.0
---

<h3>Techne v1.0.0</h3>

<strong>Welcome to techne v1.0.0 – our first official stable release for YaaS!</strong>
<p>This release focused on overall stabilization of components, patterns and clear usage guidelines and documentation. The techne page itself under techne.yaas.io is aligned with the overall builder and YaaS design and we added a lot of new design examples and guideline updates on every page! </p>

<h3>Guidelines</h3>

<h4>Visual System</h4>

<strong>Alerts, Errors & Notifications</strong>
<p>Light Blue alert color is deprected.</p>

<h4>System Errors</h4>

<strong>Generic System Errors</strong>
<p>Added a new section for System Errors to provide user-friendly guidance on error content for more generic system erros.</p>

<h3>Components</h3>

<p>New navigation structure and much more documentated techne components with clear usage guidelines and recommendations</p>

<h4>Alerts, Errors and Notifications</h4>
<p>Updated usage guidelines on alerts, errors and notifications on the components page. We limited know the amount of available colors for this pattern.</p>

<h4>Buttons</h4>
<p>We wrapped the buttons section and added a documentation for the add-something-to-a-list button</p>

<h4>Contextual Menu</h4>
<p>Documentation and guidelines for the “more” contextual menu to ise on cards and tables.</p>

<h4>Radio and Checkboxes not depricated!</h4>
<p>Custom radio and checkboxes will stay in techne.</p>

<h4>Modal boxes for dialogs</h4>
<p>Guidelines and documentation on modal boxes.</p>

<h4>Page Header</h4>
<p>Redesigned component and clear guidlines for a mobile first techne page-header.</p>

<h4>Pagination component</h4>
<p>Redesigned component for pagination with documentation and guidelines.</p>

<h4>Product Thumbnails</h4>
<p>Component for displaying products. This replaces the thumbnail component.</p>

<h4>Tables</h4>
<p>New mobile table drop behaviour for tables with interactions and/or avatars.</p>

<h4>Tour Bubble</h4>
<p>Guidelines and documentation on the tour bubble.</p>

<h4>Trees</h4>
<p>Reintroducing trees with design update.</p>

<h3>Patterns & Layouts</h3>

<p>Updated visual reference examples of how to combine techne components to design usable, intuitive consumer-grade YaaS tools. Updated overall styleguide.</p>

<h4>Cards</h4>
<p>Guidelines on icon shapes to you use on cards and tables.</p>

<p>Thank you everyone for participating and helping us to make this first major release happen.</p>
<p>There will be more in the future, stay tuned.</p>

```

Comment:
I'm not seeing any updates dealing with release notes here.  Di you forget to push?